downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -0,0 +1,45 @@
+
+[id="proc-deploy-eda-controller-with-aap-operator-ocp_{context}"]
+
+
+
+.Prerequisites
+
+* You have installed {OperatorPlatform} on {OCPShort}.
+* You have installed and configured {ControllerName}.
+
+.Procedure
+
+. Navigate to menu:Operators[Installed Operators].
+
+. Create a new Event-Driven Ansible custom resource. 
++
+This takes you to the Form View to customize your installation.
+
+. Specify your controller URL. 
++
+If you deployed automation controller in Openshift as well, you can find the URL on the Routes page.
++
+[NOTE]
+====
+This is the only required customization, but you can customize other options using the UI form or directly in the YAML configuration tab, if desired.
```

Comment:
We should find a way to document the "other options" eventually.  I think that might need a two-fold approach:
* There is a way for us to provide "in-line" documentation in the operator code that will show up in a help bar in the Openshift UI.  That way the docs are right where they will be used.
* I think for the RH docs, we should probably add a separate section or page for "Customizing your EDA Deployment" or something like that.  Then we could add things like that there.  But it might be better to use that space for capturing the "use cases/scenarios" that those advanced configuration setting would be used.  

I think that's outside the scope of this PR though, just dropping ideas here.

Reply:
Thanks for your suggestions, @rooftopcellist. Per discussion, we will create a jira to address the customizing EDA options.